opennms-base-assembly/src/main/filtered/etc/events/AlarmChangeNotifierEvents.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<events xmlns="http://xmlns.opennms.org/xsd/eventconf">
+  <event>
+    <uei>uei.opennms.org/plugin/AlarmChangeNotificationEvent/AlarmDeleted</uei>
+    <event-label>Alarm Change Notification Event : Alarm Deleted</event-label>
+    <descr>
+    &lt;p&gt;Alarm %parm[alarmid]% Deleted&lt;p&gt;
+    
+    &lt;p&gt;Deleted Alarm Values&lt;p&gt;
+    &lt;p&gt;%parm[oldalarmvalues]%&lt;p&gt;
+    </descr>
+    <logmsg dest="logndisplay">
```

Comment:
Do we need to persist these to the database? `donotpersist` may be more suitable here.